prac_05/state_names.py

```diff
@@ -0,0 +1,21 @@
+"""
+CP1404 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+print(CODE_TO_NAME)
+
+state_code = input("Enter short state: ").upper()
+while state_code != "":
+    try:
+        print(state_code, "is", CODE_TO_NAME[state_code])
```

Comment:
Consider f-string printing, though leaving the code as is would be equally as readable.

Reply:
That would make sense. This was the code we were told to copy so I didn't modify it.